api/proto/hapi/config.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+option go_package = "github.com/chanzuckerberg/happy/api/config";
+
+package happy.api.config;
```

Comment:
Can you think of a name besides `config` that would make sense here? I just want to avoid confusion with the `happy config` feature which we should probably make a proto for eventually

Reply:
Sure, will do.

Reply:
Renamed to `app`.